docs/cli.md

```diff
@@ -65,7 +65,7 @@ This command requires that a wallet is connected to the CLI...
 **`nexus tool register --on-chain <ident>`**
 
 {% hint style="warning" %}
-The specific design for onchain tools is still in progress and as a result the implementation is not yet implemented. When running the command, it will panic. 
+The specific design for onchain tools is still in progress and as a result the implementation is not yet implemented. When running the command, it will panic.
```

Comment:
"implementation is not yet implemented"

Reply:
https://github.com/Talus-Network/nexus-sdk/pull/148/commits/302c696e5adf443a46675fc53221c75dd50e199f